test/unit/printer/layers.test.js

```diff
@@ -341,6 +342,48 @@ describe('layer creation', () => {
     });
   });
 
+  describe('GeoJSON layer creation', () => {
+    const jobId = 1;
+    /** @type {Layer} */
+    const spec = {
+      type: 'GeoJSON',
+      geojson: {
+        type: 'FeatureCollection',
+        name: 'france_3857',
+        crs: {
+          type: 'name',
+          properties: { name: 'urn:ogc:def:crs:EPSG::3857' },
+        },
```

Comment:
Why put this in the fixture if it's not supported by the api ?

Reply:
good catch